scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/Hints.java

```diff
@@ -1,53 +1,97 @@
-

```

Comment:
A bunch of files in the repo have been Windows line endings up till this point... but now they are getting converted over one by one. This is much better done as its own commit fixing all of them at once, when this type of problem is noticed. Otherwise, the diff here is obfuscated by the line ending shift.